electrumx/server/daemon.py

```diff
@@ -496,3 +496,14 @@ def strip_mtp_data(self, raw_block):
         blocks = await self._send_vector('getblock', params_iterable)
         # Convert hex string to bytes
         return [hex_to_bytes(self.strip_mtp_data(block)) for block in blocks]
+
+
+class BitGreenDaemon(Daemon):
```

Comment:
Please rename this DashDaemonBase, and have DashDaemon and SmartCashDaemon inherit it.